src/main/java/io/github/oliviercailloux/y2018/apartments/apartment/Apartment.java

```diff
@@ -1,136 +1,132 @@
 package io.github.oliviercailloux.y2018.apartments.apartment;
 
+import static com.google.common.base.Preconditions.checkArgument;
+import static com.google.common.base.Preconditions.checkNotNull;
+
 import java.util.Objects;
+
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import static com.google.common.base.Preconditions.checkArgument;
-import static com.google.common.base.Preconditions.checkNotNull;
 
 public class Apartment extends Object {
 
 	private static final Logger LOGGER = LoggerFactory.getLogger(Apartment.class);
 
 	/**
-	 * @param a real number it represents the floor area of the apartment in square
-	 *          meters. Has to be initialized, if not, the apartment can't be
-	 *          created without a floor area.
+	 * A real number representing the floor area of the apartment in square meters.
+	 * Has to be initialized, if not, the apartment can't be created without a floor
+	 * area.
 	 */
 	private double floorArea;
 
 	/**
-	 * @param a String of characters that gives the full location of the apartment
-	 *          (number, street name, ZIP code, city, country), has to initialized
-	 *          Has to be initialized, if not, the apartment can't be created
-	 *          without an address.
+	 * A String of characters that gives the full location of the apartment (number,
+	 * street name, ZIP code, city, country) Has to be initialized, if not, the
+	 * apartment can't be created without an address.
 	 */
 	private String address;
 
 	/**
-	 * @param an integer superior or equal to zero, it is the number of bedrooms
-	 *           available of use in the apartment. if not initialized, the
-	 *           parameter will have a default value of <code>0</code> which mean
-	 *           the apartment doesn't have any bedroom.
+	 * An integer superior or equal to zero, it is the number of bedrooms available
+	 * of use in the apartment. If not initialized, the parameter will have a
+	 * default value of <code>0</code> which mean the apartment doesn't have any
+	 * bedroom.
```

Comment:
Indiquer ≥ 0 est utile, c’est compact, peu susceptible de changer, utile aux développeurs qui utilisent cette donnée. Par contre, le reste me semble plus dommageable qu’utile. Cette doc n’est pas compacte (et donc cache l’information plus importante), doit être maintenue à jour si la logique de construction change, est peu utile. À effacer à mon avis. OPT

Reply:
Fixed in commit 6219a6ffa7e12df6c9f8a6ef99cedafb43e61100